doc/modules/metrics.rst

```diff
@@ -33,6 +33,27 @@ the kernel:
     2. ``S = 1. / (D / np.max(D))``
 
 
+.. currentmodule:: sklearn.metrics
+
+.. _pairwise_distances:
```

Comment:
This doesn't work as a ref anchor unless followed by a heading. Not that a heading would be needed if this were more integrated into the above.

Reply:
Removed the anchor.

I thought adding the description of pairwise (kernel) distances after the introduction of distances and kernels (and how to convert them) would flow naturally (hence no header in between). Any ideas how to better integrate this part?